FirstSolution/Task4Comparer/Comparer.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Task4Germany;
+using Task4Russia;
+using Task4UnitedKindom;
```

Comment:
Check namespaces

Reply:
I've added check, but it doesn't work properly

Reply:
Everything is alright